test/Riok.Mapperly.Tests/Helpers/InlineExpressionRewriterTest.cs

```diff
@@ -20,7 +20,7 @@ public class InlineExpressionRewriterTest
     [InlineData("new TestRecord[2].ToString()", true, "new global::TestRecord[2].ToString()")]
     [InlineData("base.ToString()", false)] // CS0831
     [InlineData("(1,2).ToString()", false)] // CS8143
-    [InlineData("((string?)source)?.ToString()!", false)] // CS8072
+    [InlineData("((string)source)?.ToString()!", false)] // CS8072
```

Comment:
Could you explain this change?

Reply:
Sure,

Inline expression rewriter is used in projections, which have disabled nullable context. When a nullable type is present inside this block, the compiler is not happy (you can not use nullable annotations inside a non-nullable context).

When casts are processed in the visitor, the nullability is stripped out.

Reply:
Got it, that makes sense. But since this test has `canBeInlined=false`, there is no need to change it, is there?

Reply:
It didn't come to my mind. Now visitors are applied only when `CanBeInlined=true`.